lib/reverse_string.rb

```diff
@@ -1,5 +1,30 @@
 def reverse_characters(str)
+
+    i = 0    
+    n = str.size - 1
+    ansstr = " "*n
```

Comment:
`ansstr` is also a String the way you have it written now. The difference is that, in your code, `ansstr` is String with many spaces, whereas I'm asking how your code with change if you started out with `ansstr` as an empty String.